core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss

```diff
@@ -86,6 +87,34 @@
             margin-bottom: $spacing-3;
         }
     }
+
+    .pi-star-fill {
+        color: $color-palette-primary;
+        margin-right: $spacing-1;
+    }
+
+    .dot-pages-panel__header {
+        display: flex;
+        align-items: center;
+        margin-left: $spacing-3;
+    }
+
+    .p-panel-icons {
+        display: flex;
+        justify-content: end;
+        flex: 1;
+    }
+
+    .p-panel-header-icon {
+        .pi {
+            border: 1px solid;
+            border-radius: 100px;
+        }
+    }
+
+    .p-panel-header .p-panel-icons span {
+        margin-right: $spacing-4;
+    }
```

Comment:
We need to move this to the panel scss file and it should look like our icon button.